### DISCUSSION TITLE
How to make Eden to work with Bun Workspaces.

### DISCUSSION THREAD

**Discussion by jonyw4** [elysiajs/elysia#931] [Q&A] [answered]
Hi folks, I think this must be Typescript related but I am not able to make Eden to work using Bun Workspaces. 

You can try by converting [the example repository](https://github.com/SaltyAom/elysia-monorepo-path-alias) to workspace. You will get this after changing:

<img width="632" alt="image" src="https://github.com/user-attachments/assets/2611d461-0106-4708-8646-b558f49a8b80">

Not sure if we need to do something else on the client side to make it work

**Comment by brianorwhatever** [comment]
This error is simply telling you that you have created a const `data` but are not accessing it

**Reply by jonyw4** [reply, marked as answer]
I was able to make it work by adding `"@types/bun": "^1.1.8"` in the root level.
